scipy/sparse/linalg/_dsolve/linsolve.py

```diff
@@ -265,7 +265,9 @@ def splu(A, permc_spec=None, diag_pivot_thresh=None,
     Parameters
     ----------
     A : sparse matrix
-        Sparse matrix to factorize. Should be in CSR or CSC format.
+        Sparse matrix to factorize. Should be in CSR or CSC format. The CSC
+        format is generally more efficient than CSR. If a CSR matrix is provided,
```

Comment:
Instead of saying "If a CSR matrix...", let's be more general and say "If a different sparse format is provided..."

Reply:
It says in the previous sentence that it should be a CSC or CSR matrix. Should I remove that too?

Reply:
How about this:

Sparse matrix to factorize. Most efficient when provided in CSC format. Other formats will be converted to CSC before factorization.